R/helpers.R

```diff
@@ -144,6 +144,12 @@ do_trace <- function(current, risk, mstart,
 check_y_family <- function(y, family)
     family@check_y(y)
 
+## check function for response matrix as in FDboost
```

Comment:
This could be realised as generic function with a special `check_y_family.matrix` function and having the standard function as `check_y_family.default`